app/uploaders/league_header_image_uploader.rb

```diff
@@ -0,0 +1,56 @@
+# encoding: utf-8
+
+class LeagueHeaderImageUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # process resize_to_fit: [800, 300]
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Because of the fact that the application runs inside a docker container, it is not possible to use file system storage here. On deployment your uploads will be gone. 
The system of choice is Amazon S3 right now. With `storage :fog` it's quite easy to use.